lms/config/resources.py

```diff
@@ -220,12 +221,10 @@ def provisioning_enabled(self):
         :raise HTTPBadRequest: if there's no oauth_consumer_key in the request
           params
         """
-        application_instance = find_by_oauth_consumer_key(
-            self._request.db, self._get_param("oauth_consumer_key")
-        )
-        if application_instance is None:
+        try:
+            return self._ai_getter.provisioning(self._get_param("oauth_consumer_key"))
+        except ConsumerKeyError:
```

Comment:
The `provisioning` method already handles `ConsumerKeyError`, so I don't think you need to do that here?

Reply:
Ah yes, this functionality moved into the ai_getter service. Fixed